### DISCUSSION TITLE
Cameras taken offline by constant requests.

### DISCUSSION THREAD

**Discussion by dracov2** [mrlt8/docker-wyze-bridge#329] [Q&A]
Hello,

Brand new setup on Pi 3 b+, running newest home assistant.
Got 1/13 wyze cams updated to allow rtsp, called it OfficeCam.
Plugged in my Wyze credentials to Docker Wyze Bridge, got it started and after a min or so I get all 13 cameras not responding within the Wyze App.

As soon as I stop Docker Wyze Bridge everything goes back online withing the Wyze App. 
It feels kinda like its overflooding with requests and it takes the cams offline.

I followed https://www.youtube.com/watch?v=Mxpwh5I_pUs to setup the stream.

Logs:
![image](https://user-images.githubusercontent.com/102422602/160244467-8ce335a9-6226-41e1-88d7-1c6863183e5b.png)

1/50 refreshes of the Overview of the Home Assistant. I am able to get the preview of the OfficeCam.
![image](https://user-images.githubusercontent.com/102422602/160244650-7eb3dea8-05c5-43c0-b084-b3d3e6a964da.png)

Any idea what could be causing it?
Do I have to have all of the cams flashed to accept rtsp?

Thanks!

**Comment by mrlt8** [comment]
Is this v1.2.2? There was bug in v1.2.1 that was causing some memory issues with cams that were offline.

The bridge should work with all firmware versions, so you shouldn't need to flash.

Depending on your wifi, 13 cams could be causing your wifi utilization to max out..

Can you try adding a few cameras at a time to `FILTER_NAMES` e.g. `sc, living room cam, gym` (may have to toggle "Show unused optional configuration options")

Or you could also try setting `FILTER_MODELS` to `v2,v3` to see if that helps limit the number of cams and also avoid draining the battery on the outdoor models.

**Reply by dracov2** [reply]
I am on v1.2.2.

For my wifi Im using Netgear Orbi AX4200 Mesh with 2 additional satellites. 

Based on your suggestion I have added Filter_names just for the OfficeCam and it appears to be helping. 
I am able to get the video stream quite reliably. It is delayed by about 4 secs when compared to the wyzeApp. Not sure if that is normal or not.

I will start flashing other cams with the RTSP firmware and will slowly add them on.

**Reply by mrlt8** [reply]
Unfortunately, the bridge does seem to be about a second behind the wyze app.


The RTSP firmware is not required for the bridge, but if you're already on an rtsp firmware, you could probably skip the bridge and connect directly to the rtsp stream on the camera.

**Comment by NotaProgrammer26** [comment]
Just set up my 19 Wyze Cameras with the bridge.   My Raspberry PI 4 went to 100% pretty quickly.  Had lots of issues and cameras dropping and Wyze saying they were off line.    Filtered out all but 7 cameras and set the RTSP to 15.   Then the cameras connected and stayed connected.   My Raspberry PI stayed at 24% for the AddOn.  I was also able to run the HLS of two of the cameras on my PC at the same time with no impacts to Wyze or the Raspberry PI.  Using this format:  http://homeassistantrpi4.local:8888/oregon-camera/  (put in your HA name and your camera name into the URL.  )     So my recommendation is to "filter" out the cameras you don't need, use a lower RTSP 15 or 30 as the default.   Then go from there.    There does appear to be a delay on the streamed time from the actual time.   Sometimes a few minutes.  Other times a few seconds.     P.S. I'm running default Wzye firmware on the cameras.

**Reply by mrlt8** [reply]
> My Raspberry PI 4 went to 100% pretty quickly. 

I believe the pi 4 has 4 cores, so 100% should be roughly equivalent to 25% of total CPU usage?

> set the RTSP to 15.

Is that for the snapshots? If it is, a lower value will take more frequent snapshots from the stream and could potentially use more resources since FFMPEG will have to decode the stream for each snapshot. Disabling RTMP and/or any other unused streams might also help reduce usage.

As for the delay, I believe the HLS stream will have the most delay, so you should try to use RTSP if possible.

**Reply by ddenson** [reply]
> So my recommendation is to "filter" out the cameras you don't need, use a lower RTSP 15 or 30 as the default.

How did you "filter" out the cameras? I would like to remove my outdoor cameras as the batteries are drained pretty quickly (less than 8 hrs). Thanks in advance!

**Reply by mrlt8** [reply]
something like:


```yaml
    - FILTER_MODELS=WVOD1
    - FILTER_BLOCK=true
```

https://github.com/mrlt8/docker-wyze-bridge/wiki/Camera-Filtering

**Reply by ddenson** [reply]
Adding the following to the configuration worked, thank you!

FILTER_NAMES: backyard
FILTER_BLOCK: true
